ingestion/src/metadata/ingestion/source/database/athena/metadata.py

```diff
@@ -142,16 +136,30 @@ def get_columns(self, connection, table_name, schema=None, **kw):
             "default": None,
             "autoincrement": False,
             "comment": c.comment,
-            "raw_data_type": c.type if is_complex(c.type) else None,
+            "raw_data_type": c.type,
+            "is_complex": is_complex_type(c.type),
             "dialect_options": {"awsathena_partition": True},
         }
         for c in metadata.partition_keys
     ]
     return columns
 
 
+@reflection.cache
+def get_view_definition(self, connection, view_name, schema=None, **kw):
+    """
+    Gets the view definition
+    """
+    full_view_name = view_name if not schema else f"{schema}.{view_name}"
+    res = connection.execute(f"SHOW CREATE VIEW {full_view_name}").fetchall()
+    if res:
+        return "\n".join(i[0] for i in res)
```

Comment:
is this a list of lists?  and are we sure I[0] always available?

Reply:
yes i[0] is always available

basically if your view definition is like 
```
create view 
as 
select * from table
```

it will return as [(create view ,), (as,),(select * from table,]